hedera-mirror-rest/utils.js

```diff
@@ -633,10 +636,16 @@ const returnEntriesLimit = (type) => {
 /**
  * Converts the byte array returned by SQL queries into hex string
  * @param {Array} byteArray Array of bytes to be converted to hex string
- * @return {hexString} Converted hex string
+ * @param {boolean} addPrefix Whether to add the '0x' prefix to the hex string
+ * @return {String} Converted hex string
  */
-const toHexString = (byteArray) => {
-  return byteArray === null ? null : byteArray.reduce((output, elem) => output + `0${elem.toString(16)}`.slice(-2), '');
+const toHexString = (byteArray, addPrefix = false) => {
+  if (byteArray === null) {
```

Comment:
nit: use `isnil` for coverage of undefined also
```suggestion
  if (_.isnil(byteArray)) {
```